features/invoicing.feature

```diff
@@ -0,0 +1,318 @@
+    And the user "admin" sends a "GET" request to "/api/invoice_line_items?store=1"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the JSON should match:
+      """
+      {
+        "@context":"/api/contexts/Order",
+        "@id":"/api/orders",
+        "@type":"hydra:Collection",
+        "hydra:member":@array@,
+        "hydra:totalItems":259,
+        "hydra:view":{
+          "@id":"/api/invoice_line_items?store=1\u0026page=1",
+          "@type":"hydra:PartialCollectionView",
+          "hydra:first":"/api/invoice_line_items?store=1\u0026page=1",
+          "hydra:last":"/api/invoice_line_items?store=1\u0026page=9",
+          "hydra:next":"/api/invoice_line_items?store=1\u0026page=2"
+        },
+        "hydra:search":{
+          "@type":"hydra:IriTemplate",
+          "hydra:template":"/api/invoice_line_items{?date,state,state[],exists[exports],store,store[]}",
+          "hydra:variableRepresentation":"BasicRepresentation",
+          "hydra:mapping":[
+            {
+              "@type":"IriTemplateMapping",
+              "variable":"date",
+              "property":"date",
+              "required":false
+            },
+            {
+              "@type":"IriTemplateMapping",
+              "variable":"state",
+              "property":"state",
+              "required":false
+            },
+            {
+              "@type":"IriTemplateMapping",
+              "variable":"state[]",
+              "property":"state",
+              "required":false
+            },
+            {
+              "@type":"IriTemplateMapping",
+              "variable":"exists[exports]",
+              "property":"exports",
+              "required":false
+            },
+            {
+              "@type":"IriTemplateMapping",
+              "variable":"store",
+              "property":"store",
+              "required":false
+            },
+            {
+              "@type":"IriTemplateMapping",
+              "variable":"store[]",
+              "property":"store",
+              "required":false
+            }
+          ]
+        }
+      }
+      """
+
+  Scenario: Get invoice line items filtered by multiple stores
+    Given the fixtures files are loaded:
+      | cypress://setup.yml |
+    Given the fixtures files are loaded with no purge:
+      | cypress://package_delivery_orders.yml |
+    Given the user "admin" is authenticated
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I add "Accept" header equal to "application/ld+json"
+    And the user "admin" sends a "GET" request to "/api/invoice_line_items?store[]=1&store[]=2"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the JSON should match:
+      """
+      {
+        "@context":"/api/contexts/Order",
+        "@id":"/api/orders",
+        "@type":"hydra:Collection",
+        "hydra:member":@array@,
+        "hydra:totalItems":272,
```

Comment:
--- Failed steps:

001 Scenario: Get invoice line items filtered by multiple stores # features/invoicing.feature:164
      And the JSON should match:                                 # features/invoicing.feature:175
        Expected JSON doesn't match response JSON.
        Value "280" does not match pattern "272" at path: "[hydra:totalItems]" (RuntimeException)